pkg/operation/botanist/component/kubeapiserver/kube_apiserver.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubeapiserver
+
+const (
+	// Port is the port exposed by the kube-apiserver.
+	Port = 443
```

Comment:
If the port is exposed anyway, why not use this variable for example also in kube_apiserver_service.go?

Reply:
Hm, I have used it in `kube_apiserver_service.go`. Am I missing something? Can you point to the concrete places where you suggest to use it?

Reply:
See the reference below.